database/src/main/postgres/runs/V0.3.0.1__get_ancestors.sql

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+CREATE OR REPLACE FUNCTION runs.get_ancestors(
+    IN i_id_partitioning    BIGINT,
+    IN i_limit              INT DEFAULT 5,
+    IN i_offset             BIGINT DEFAULT 0,
+    OUT status              INTEGER,
+    OUT status_text         TEXT,
+    OUT ancestorid         BIGINT,
+    OUT partitioning        JSONB,
+    OUT author              TEXT,
+    OUT has_more            BOOLEAN
+)  RETURNS SETOF record AS
+$$
+    -------------------------------------------------------------------------------
+--
+-- Function: runs.get_ancestors(3)
```

Comment:
Isn't this out of the scope of this PR?

Besides, isn't it implemented here: https://github.com/AbsaOSS/atum-service/pull/311/files

also this PR has unresolved merge conflicts, thus I don't think it's ready for the review so far. If you need any help or clarification I'm happy to assist you though!